plugins/kotlin-dataframe/README.md

```diff
@@ -0,0 +1,10 @@
+## :plugins:kotlin-dataframe
+
+This module, published as "compiler-plugin-all", holds the Kotlin 2.x Compiler plugin of DataFrame.
+
+A [work-in-progress](https://github.com/Kotlin/dataframe/issues/704)
+plugin for your Gradle project that can generate on-the-fly column accessors for the compiler and IDE even without
+having to provide data schemas!
+
+Details of how to build and run this for yourself will follow, but there is a
```

Comment:
What do you mean by build? publish to maven local and use local build?

Reply:
I think I meant "details of how to use it in your own project"... but I'm not sure, I'll rephrase it